jvcl/run/JvOfficeColorPanel.pas

```diff
@@ -115,7 +115,7 @@   TJvColorSpeedButton = class(TJvSpeedButton)
     property DrawColor: TColor read FDrawColor write SetDrawColor default clDefault;
     property DisabledDrawColor: TColor read FDisabledDrawColor write SetDisabledDrawColor default clGray;
     property EdgeWidth: Integer read GetEdgeWidth write SetEdgeWidth;
-    property OnEnabledChagned: TNotifyEvent read FOnEnabledChagned write FOnEnabledChagned;
+    property OnEnabledChanged: TNotifyEvent read FOnEnabledChanged write FOnEnabledChanged;
```

Comment:
This is a breking changing.

I would go for it, but maybe Andreas or Oliver also have an opinion

Reply:
Yes, I'm not fan of these, there must be someone somewhere who published it and it will break their code.
Now, we could have a deprecated interposing property with getters and setters but that might be an overkill.
In the end, I'm in favor of adding a comment saying "yes we know about the typo but we have to keep it"

Reply:
If someone publishes the the event with the typo in a derived class, a recompile of that derived class would show the compile error. And the unit with the derived class must be recompiled anyway if the JVCL is updated.

A possible DFM streaming problem (wrong event name in the DFM) can only come from the derived class, so that problem would fall into the responsibility of the derived class. In other words the JVCL would be fine because the typo  in the event property was never "published" by the JVCL itself.

So I don't see why we can't fix that typo.

Reply:
Ok,but what is now conclusion?
I can live with both, would prefer to do it.

@obones : I would say: Take the decision, you are the longest and oldest with the project (I think)

Reply:
As I see this, that fixing a typo is step forward. I would not mind fixing my own code, if typo is fixed in library I am using.

Reply:
Ok, considering the fact that most users will never derive their own classes from ours I changed my mind and decide that this renaming should be applied.